src/utils/helpers.js

```diff
@@ -0,0 +1,30 @@
+export function SetButtonText(
+  isLoading,
+  btn,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    btn.textContent = loadingText;
+  } else {
+    btn.textContent = defaultText;
+  }
+}
+
+export function handleAddCardSubmit(request, evt, loadingText = "Saving...") {
```

Comment:
`handleAddCardSubmit` should be named neutrally: `handleSubmit`. It can be used for any form submit